### PR TITLE
Add tests for BI modulations

### PR DESCRIPTION
**Pull request by mohierf**

### CODE REVIEW

File: alignak/objects/businessimpactmodulation.py [hunk 1]
```diff
@@ -76,7 +76,25 @@ def get_name(self):
...
+
+        * Call our parent class is_correct checker
+
+        :return: True if the configuration is correct, otherwise False
+        :rtype: bool
+        """
+
+        # Ok just put None as modulation_period, means 24x7
+        if not hasattr(self, 'modulation_period'):
+            self.modulation_period = None
```
Comment (ddurieux) [inline alignak/objects/businessimpactmodulation.py]: It's not the best place to do this, so manage it in __init__ or perhaps if has defaut_fill or something like it (don't remember the function)

Comment (mohierf) [inline alignak/objects/businessimpactmodulation.py]: It is done in this function for the SchedulingItem ... this is why I did here. I will move this test to hte __init__

File: alignak/objects/businessimpactmodulation.py [hunk 2]
```diff
@@ -70,13 +70,22 @@ class Businessimpactmodulation(Item):
                        'modulation_period':               StringProp(default=''),
                        })
 
+    def __init__(self, params=None, parsing=True):
+        super(Businessimpactmodulation, self).__init__(params, parsing=parsing)
+
+        # Ok just put None as modulation_period, means 24x7
+        if not hasattr(self, 'modulation_period'):
+            self.modulation_period = '24x7'
```
Comment (Seb-Solon) [inline alignak/objects/businessimpactmodulation.py]: Shouldn't this be an uuid?

Comment (mohierf) [inline alignak/objects/businessimpactmodulation.py]: I will have a look to this but it seems that BI modulation have some specific behavior ...

**Pull request merged**